### ISSUE TITLE
Choose Instances to Track in Columns (Custom Timelines)

### ISSUE THREAD

**Issue by ryanleesipes** [tootsuite/mastodon#1207]
## Feature Request!

The idea was submitted today that as new instances pop up with different content rules and focuses, that a good feature to have would be to add those instances as timelines.

For instance, I may have my local timeline, the federated timeline, the Ubuntu timeline (once they launch an instance), the Awoo timeline, and so on. If I only want to the view the Ubuntu timeline I can add that and pick that as an option.

**TLDR:** Add ability to create custom timelines that point at other instances.

* * * *

-  [Yes] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by ryanleesipes** [comment]
Whoops! Finally found this in  #1053
